defaults/preferences/prefs.js

```diff
@@ -15,6 +15,8 @@
 // App-specific preferences, in alphabetical order.
 
 pref('browser.dom.window.dump.enabled', true);
+pref('browser.hiddenWindowChromeURL', 'chrome://qbrt/content/main.html');
```

Comment:
just so I'm clear: qbrt's shell is always going to live at `chrome/content/main.html`, and this URL ought to never change (without devs doing something funny)?